packages/shared/routes/dashboard/Dashboard.svelte

```diff
@@ -156,11 +163,11 @@
                 dashboardRoute.set(tab)
             }
         }
-        if (!$appSettings.deepLinking) {
+        if (mounted && !$appSettings.deepLinking) {
```

Comment:
Why is this check for mounting needed? In case accounts load before it mounts?

Reply:
I think this is an artifact from when there was multiple resource leaks and I was getting a lot of unexpected behaviour. I will try to think if this is needed now and test if we remove it.